go/arrow/flight/server.go

```diff
@@ -49,9 +49,36 @@ type (
 	Action                          = flight.Action
 	ActionType                      = flight.ActionType
 	Result                          = flight.Result
+	CancelFlightInfoResult    = flight.CancelFlightInfoResult
+	CancelResult = flight.CancelResult
```

Comment:
indentation

Reply:
Oh, sorry.
I've ran `go fmt arrow/flight/server.go` in `go/`.

BTW, I got many change when I run `go fmt ./...`. What command do you use for formatting?